roles/st2/tasks/2.version.yml

```diff
@@ -32,6 +32,12 @@
   when: st2_revision == 'current'
   tags: [st2, version]
 
+- name: version | Set which StackStorm revision to install manually
+  set_fact:
+    _st2_revision: {{ st2_revision }}
```

Comment:
From CircleCI build https://circleci.com/gh/StackStorm/ansible-st2/24:

```
ERROR: Syntax Error while loading YAML script, /home/ubuntu/ansible-st2/roles/st2/tasks/2.version.yml
Note: The error may actually appear before this position: line 37, column 21

  set_fact:
    _st2_revision: {{ st2_revision }}
                    ^
We could be wrong, but this one looks like it might be an issue with
missing quotes.  Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"
```

Pretty awesome error handling from Ansible here :)

So we need just:
```yaml
_st2_revision: "{{ st2_revision }}"
```